integration/lifecycle/lifecycle_test.go

```diff
@@ -44,6 +44,20 @@ var _ = Describe("Creating a container", func() {
 		})
 	})
 
+	Describe("Docker image download in a container", func() {
+		It("returns a helpful error message when image not found from default registry (#89007566)", func() {
+			client = startGarden()
+			_, err := client.Create(garden.ContainerSpec{RootFSPath: "docker:///cloudfoundry/doesnotexist"})
+			Expect(err.Error()).To(ContainSubstring("could not fetch: cloudfoundry/doesnotexist image from https://index.docker.io/v1/ registry"))
+		})
+
+		It("returns a helpful error message when image not found from custom registry (#89007566)", func() {
+			client = startGarden()
+			_, err := client.Create(garden.ContainerSpec{RootFSPath: "docker://example.com/cloudfoundry/doesnotexist"})
+			Expect(err.Error()).To(ContainSubstring("could not fetch: cloudfoundry/doesnotexist image from example.com registry"))
```

Comment:
The intention was to include additional error information from the underlying failure.
If this is not Ok, please suggest so and I will update.